### ISSUE TITLE
twisted.application.app.initialLog formats the reactor class poorly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3167]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3167|
|Type|enhancement|
|Created|2008-04-05 16:14:37Z|

It does it like this:

```
2008-04-05 12:12:39-0400 [-] reactor class: <class 'twisted.internet.selectreactor.SelectReactor'>
```

Having the Python FQPN or a human-readable name (like `select`) would be an improvement.

<details><summary>Searchable metadata</summary>

```
trac-id__3167 3167
type__enhancement enhancement
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1207412077000000 1207412077000000
changetime__1213837903000000 1213837903000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

This was fixed as part of [#3052](https://github.com/twisted/twisted/issues/3052) in r23738

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
